app/.yarnclean

```diff
@@ -0,0 +1,39 @@
+# test directories
```

Comment:
Why are we using yarn instead of npm here? Do we have to migrate to yarn? I mean, I dont want to use npm and yarn side by side as they do mostly the same thing.

Reply:
@wdanilo it came with Electron's quickstart repo. I am cleaning everything up now